hw/ip/aes/data/aes.hjson

```diff
@@ -498,18 +499,21 @@
             desc: '''
               3'b001: Reseed the masking PRNG once per block.
               Invalid input values, i.e., values with multiple bits set and value 3'b000 are mapped to PER_1 (3'b001).
+              This results in a max entropy consumption rate of ~286 Mbit/s.
             '''
           },
           { value: "2",
             name: "PER_64",
             desc: '''
               3'b010: Reseed the masking PRNG approximately once per every 64 blocks.
+              This results in a max entropy consumption rate of ~4.5 Mbit/s.
             '''
           },
           { value: "4",
             name: "PER_8K",
             desc: '''
               3'b100: Reseed the masking PRNG approximately once per every 8192 blocks.
+              This results in an max entropy consumption rate of ~0.035 Mbit/s.
```

Comment:
Great, thanks for adding this!
Just a nit that we may have to specify the clock frequency assumed, since in another system this might be different.